src/containers/Layout/Layout.tsx

```diff
@@ -154,6 +156,8 @@ const Layout = () => {
           currentProfile={profile ?? undefined}
```

Comment:
There are a lot of profile properties added here. Can we combine them in a single object to make passing around cleaner?

Reply:
https://github.com/jwplayer/ott-web-app/pull/371/commits/df8266634ac7eaba18352129b93fdf5d38761a25